src/app/donation-start/donation-start-form/donation-start-form-parent.component.ts

```diff
@@ -1481,6 +1488,9 @@ export class DonationStartFormParentComponent implements AfterContentChecked, Af
     this.stripeResponseErrorCode = undefined;
 
     this.stripePaymentMethodReady = false;
+    if (this.stripeSavedMethods.length < 1) {
+      this.paymentMethodChecked = false;
```

Comment:
wouldn't `this.paymentMethodChecked` have to already be false if there are no saved methods?